### DISCUSSION TITLE
MUTUAL

### DISCUSSION THREAD

**Discussion by dblodorn** [public-assembly/.github#1] [Ideas]
Let's talk about this meta design system!

**Comment by salieflewis** [comment]
Neesh and I spoke earlier about kicking off this process.

I'm tempted to lean on existing design systems that are extensible and focus efforts on building out the design interactions element of MUTUAL in a nuanced way.

How did you guys come to the decision to build on top of Vanilla Extract for Zord? Having put a lot of energy into a similar thing recently, what would you say about the technical implementation of the design system? How did it come about, where has it been useful, where has it lacked?

**Reply by ioeylim** [reply]
are you referring to notion when you say existing design systems? @salieflewis

**Reply by salieflewis** [reply]
No. Existing design systems outside of anything we have done.

**Comment by Valcoholics** [comment]
Considering using the PA onchain theme that I made following the Mutual Design System or the documentation site for Flexible docs

**Comment by salieflewis** [comment]
@losingmyego Care to provide any type of up to date thinking here?

Is this a forum you'd be interested in working on MUTUAL via?

**Reply by losingmyego** [reply]
Oh wow I've never seen this thread! I currently have an active Notion for Mutual. 

Not sure if it makes sense or not to utilize a GitHub forum for discussion, I'm unfamiliar with using it as a forum tool.  

I'm probably interested in keeping it in one place, but very open to discuss thoughts

**Reply by salieflewis** [reply]
Given our last conversation, I think it makes sense to keep it on Notion. I think in the short future there will be a design system repository pop up here, and we can use GitHub discussions around that repo to communicate about Mutual.
